dockerfiles/CohortVcfToZarr/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3.7.2
+
+RUN pip install --upgrade pip
+
+COPY requirements.txt .
+RUN pip3 install numpy==1.19.1
+RUN pip3 install zarr==2.4.0
+RUN pip3 install scikit-allel==1.3.1
```

Small suggestion but if you are copying the requirements file into the docker image then you could just do this.
```suggestion
COPY requirements.txt .
RUN pip3 install -r requirements.txt
```

---

I like it!